packages/forms-web-app/src/controllers/projects/documents/utils/documents/documents-view-model.js

```diff
@@ -1,17 +1,21 @@
 const { formatDate } = require('../../../../../utils/date-utils');
+
+const formatDocumentToViewModel = (document) => ({
```

Comment:
Good improvement but unnecessary to spend time on this and adding an extra thing to look at on the PR.